### ISSUE TITLE
Nested views - page parameter breaks parent view

### ISSUE THREAD

**Issue by Skipper-is** [farmOS/farmOS#343]
Using FarmOS 1.5
This is from /farm/area/winters-field
![image](https://user-images.githubusercontent.com/28678069/91606374-db66b280-e969-11ea-9c79-9f9c2eb96992.png)
All the logs, assets etc are in place. My history of assets drop down goes onto multiple pages, so if I go onto page 2:
url is farm/area/winters-field?page=1
![image](https://user-images.githubusercontent.com/28678069/91606492-110b9b80-e96a-11ea-9dda-2b0fa353aa87.png)
All the other drop down views are empty. The history of assets is correctly on page 2, but the rest is gone.

**Comment by mstenta** [comment]
Ah yea - this is because all those Views are essentially embedded in the one page, so they are all sharing the same GET parameters. In other words: when `?page=1` is on the URL, ALL the Views try to show that page... it is not limited to the View that you clicked the pager link from.

So that explains why it works that way... next step is to figure out how we can improve on that behavior to make it less confusing.

One potential approach would be to remove the pager entirely and just have a "More" link, which brought you to a dedicated page for each View (taking you off of the Area page itself). This would require changes in a lot of different places, however, because right now those Views are sharing the same display in all contexts, so we would need to create a second display to override the pager and remove it.

Another option might be to enable Ajax on all the Views - so that clicking the pager would refresh the View without reloading the page. However, I made the intentional decision to avoid using Ajax in Views in farmOS early on because I found that it caused issues in some cases. I don't recall exactly where, but I'm hesitant to turn it back on. Also, it may not actually solve the issue. It might refresh all the Views on the page... not sure. So I'm not really in favor of this approach.
